modules/nw-ovn-ipsec-disable.adoc

```diff
@@ -8,6 +8,11 @@
 
 As a cluster administrator, you can disable IPsec encryption only if you enabled IPsec after cluster installation.
 
+[IMPORTANT]
+====
+To avoid issues with your installed cluster after you disable IPsec, ensure that you delete the associated IPsec daemonsets.
+====
```

Comment:
The manual removal of IPsec daemonset must be done only after IPsec disable are completely rolled out onto the cluster.  this can be confirmed by running following command on each node and output must be false and and network and other cluster operators to settle down.

`$ oc -n openshift-ovn-kubernetes rsh ovnkube-node-<XXXXX> ovn-nbctl --no-leader-only get nb_global . ipsec`